ansible/lookup_plugins/bitwarden.py

```diff
@@ -0,0 +1,218 @@
+# (c) 2022 Johnathan Kupferer
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
```

Comment:
running `ansible-doc  -t lookup bitwarden -M lookup_plugins/` shows that only the `_terms` are required.

Then if you run the module, it fails with:

```
TASK [debug] *************************************************
Tuesday 15 February 2022  11:06:41 +0100 (0:00:00.011)       0:00:00.011 ******
fatal: [localhost]: FAILED! => {"msg": "An unhandled exception occurred while running the lookup plugin 'bitwarden'. Error was a <class 'ansible.errors.AnsibleError'>, original message: Bitwarden is unauthenticated and one of client_id, client_secret, or master_password were not provided. Bitwarden is unauthenticated and one of client_id, client_secret, or master_password were not provided"}
```

Should `client_id`, `client_secret`, and `master_password` have `required: True` too?

Or maybe the error message should specify the alternative to use BW_SESSION (and get it using `bw login` beforhand + export BW_SESSION=...) ?

When i execute the `bw login` command, and then export the BW_SESSION, then run the following playbook:


```yaml
- hosts: localhost
  connection: local
  gather_facts: false
  tasks:
    - debug:
        msg: "{{ lookup('bitwarden', 'A secret that exists in bitwarden').login }}"
```

I get the error:

```
TASK [debug] **********************************************************************************************************************
Tuesday 15 February 2022  11:10:36 +0100 (0:00:00.011)       0:00:00.011 ******
fatal: [localhost]: FAILED! => {"msg": "An unhandled exception occurred while running the lookup plugin 'bitwarden'. Error was a <class 'AttributeError'>, original message: 'BitwardenCLI' object has no attribute 'bitwardencli_appdata_dir'. 'BitwardenCLI' object has no attribute 'bitwardencli_appdata_dir'"}
```